app/models/candidates/verification_code.rb

```diff
@@ -35,7 +35,12 @@ def exchange
 
     request = GetIntoTeachingApiClient::ExistingCandidateRequest.new(identity_data)
     api = GetIntoTeachingApiClient::SchoolsExperienceApi.new
-    api.exchange_access_token_for_schools_experience_sign_up(code, request)
+    existing_sign_up = api.exchange_access_token_for_schools_experience_sign_up(code, request)
+
+    existing_sign_up.tap do |info|
```

Comment:
Do we not just always want to overwrite the `existing_sign_up` with what the user has entered?

Reply:
@ethax-ross only when it's missing in the CRM.